pkg/scheduler/generic_scheduler.go

```diff
@@ -421,28 +421,27 @@ func prioritizeNodes(
 	}
 
 	// Run the Score plugins.
-	scoresMap, scoreStatus := fwk.RunScorePlugins(ctx, state, pod, nodes)
+	allPluginScores, scoreStatus := fwk.RunScorePlugins(ctx, state, pod, nodes)
 	if !scoreStatus.IsSuccess() {
 		return nil, scoreStatus.AsError()
 	}
 
 	// Additional details logged at level 10 if enabled.
 	klogV := klog.V(10)
 	if klogV.Enabled() {
-		for plugin, nodeScoreList := range scoresMap {
-			for _, nodeScore := range nodeScoreList {
-				klogV.InfoS("Plugin scored node for pod", "pod", klog.KObj(pod), "plugin", plugin, "node", nodeScore.Name, "score", nodeScore.Score)
+		for _, pluginScores := range allPluginScores {
+			for _, score := range pluginScores.Scores {
+				klogV.InfoS("Plugin scored node for pod", "pod", klog.KObj(pod), "plugin", score.Name, "node", pluginScores.Name, "score", score.Score)
```

Comment:
suggest to update as this,  nothing major but jut make the code a little bit more readable,

```
		for _, nodeScores := range nodesScores {
			for _, pluginScore := range nodeScores.Scores {
				klogV.InfoS("Plugin scored node for pod", "pod", klog.KObj(pod), "node", nodeScores.Name, "plugin", pluginScore.Name, "score", pluginScore.Score)
```